### ISSUE TITLE
nx language server is throwing error when path defined with an asterisk

### ISSUE THREAD

**Issue by Tallyb** [nrwl/nx-console#2222]
## Current Behavior
Tons of errors like this one are being thrown by the language server, when a path is defined with an asterisk. 
[{
	"resource": "/Users/tally/dev/nx-tsconfig/app1/tsconfig.json",
	"owner": "typescript",
	"severity": 8,
	"message": "File '/Users/tally/dev/no-nx-tsconfig/lib1/src/*' not found.\n  The file is in the program because:\n    Root file specified for compilation",
	"source": "ts",
	"startLineNumber": 1,
	"startColumn": 1,
	"endLineNumber": 1,
	"endColumn": 2
}]

the paths in tsconfig.base are defined as follow: 
```
    "paths": {
      "lib1/*": ["lib1/src/*"],
      "lib2/*": ["lib2/src/*"]
    }
```
## Expected Behavior
Path with Asterisk is valid and should not throw error. 

## Steps to Reproduce

https://github.com/Tallyb/nx-tsconfig
npm install. 
Select any ts file. wait few seconds. Errors like bellow will show up. 
Disabling the nx extension will remove the errors. 

### Failure Logs / Images / Videos

![Screenshot 2024-07-29 at 20 07 13](https://github.com/user-attachments/assets/94742c64-9c20-482d-8ff1-f46187cfc153)

### Environment

- Nx Console version: 18.25
- VSCode version: 1.91.1

**Comment by MaxKless** [comment]
Thanks!! I just learned about this recently and the asterisk is the missing piece in the config that will create this error. 
I'll fix it asap, thanks!

**Comment by MaxKless** [comment]
It should be fixed with https://github.com/nrwl/nx-console/pull/2224

**Comment by MaxKless** [comment]
Thanks again for the report! Fixed with #2224, I'll cut a release today
